### ISSUE TITLE
Nuxt.js and vite

### ISSUE THREAD

**Issue by mdbetancourt** [nuxt/nuxt#7362]
### What problem does this feature solve?

What do you think about the new vite project https://github.com/vuejs/vite, when vite get production ready could we use it instead of webpack for development

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c10654">#c10654</a>)</em></sub></div>

**Comment by Atinux** [comment]
Hi @mdbetancourt 

Right now we keep the focus on Vue 3 + Webpack 5 since it keeps the same developer experience both in development as in production (+ all the advantages of using Webpack).

We are already keeping an eye on [vite](https://github.com/vuejs/vite) as well as [zipe](https://github.com/pikax/zipe) to see how we can also support this solution, but it will be only when we are sure that the developer experience is not compromised.

**Comment by richardeschloss** [comment]
I think there are some really great things with Webpack + Vue 3. I was curious to try Vite, and yes, while it is really cool, I realized it still has that "experimental feel" to it, and might be like that for a while. However, if anyone's interested, I just pushed ["nuxt-vite"](https://github.com/richardeschloss/nuxt-vite) to my repo. It's nothing really special, because all it is is symlinking the main Nuxt folders. But it can be a useful thing to have for a developer (one screen has Nuxt, the other Vite. So sometimes is Nuxt app server restarts, while it's restarting, the dev would still be able to make progress on the client side and see those changes).
